test/sslapitest.c

```diff
@@ -3740,9 +3747,13 @@ static int test_ciphersuite_change(void)
     const SSL_CIPHER *aes_128_gcm_sha256 = NULL;
 
     /* Create a session based on SHA-256 */
-    if (!TEST_true(create_ssl_ctx_pair(TLS_server_method(), TLS_client_method(),
-                                       TLS1_VERSION, 0,
+    if (!TEST_true(create_ssl_ctx_pair(libctx, TLS_server_method(),
+                                       TLS_client_method(), TLS1_VERSION, 0,
                                        &sctx, &cctx, cert, privkey))
+            || !TEST_true(SSL_CTX_set_ciphersuites(sctx,
+                                                   "TLS_AES_128_GCM_SHA256:"
+                                                   "TLS_AES_256_GCM_SHA384:"
+                                                   "TLS_AES_128_CCM_SHA256"))
```

Comment:
Are these extra cipher suites needed here?

Reply:
They're not extra -- this is fewer than the default list (e.g., chacha/poly is excluded)

Reply:
This test is about resuming a session where the ciphersuite changes. In TLSv1.3 this is allowed, as long as the hash remains the same. So we need 2 ciphersuites available which both support SHA256. I also wanted one other non-SHA256 ciphersuite. We need to explicitly configure these because the CCM one is not available by default.